### ISSUE TITLE
MD037: False positive with LaTex Formulars using multiple subscripts

### ISSUE THREAD

**Issue by Knalltuete5000** [DavidAnson/markdownlint#597]
With the usage of LaTex formulars in markdown text and multiple subscripts in the formular a false positive of the [`MD037- Spaces inside emphasis markers`](https://github.com/DavidAnson/markdownlint/blob/v0.26.2/doc/Rules.md#md037---spaces-inside-emphasis-markers)  can be created with the usage of additional _italic_ words.

Following is an example for creating a false-positive under the describe circumstances:

```md
$E_{K_1}(M) = C$ using _ParamName_ $M$ creating _Result_ $C$ with usage of $K_1$
```

This issue was found in the VS-Code extension version v0.48.1

**Comment by DavidAnson** [comment]
FYI, the $$ syntax is not part of Markdown. Support for that is provided in the extension via the texmath plugin. However, rules that don't use the parser won't benefit from that. I'll have a look.

**Comment by zabbarob** [comment]
Since [GitHub also supports math in Markdown](https://github.blog/2022-05-19-math-support-in-markdown/), I'd appreciate if math support could be enabled :)

**Comment by SandroMartens** [comment]
Same issue here, but with multiple multiplication marks:
`$ x * y * z$`

**Comment by mimoo** [comment]
same issue here
